Realm/RLMManagedArray.mm

```diff
@@ -227,7 +238,8 @@ - (void)insertObject:(id)object atIndex:(NSUInteger)index {
 - (void)insertObjects:(id<NSFastEnumeration>)objects atIndexes:(NSIndexSet *)indexes {
     changeArray(self, NSKeyValueChangeInsertion, indexes, ^{
         NSUInteger index = [indexes firstIndex];
-        RLMAccessorContext context(*_objectInfo);
+        RLMAccessorContext context(*_ownerInfo, *_objectInfo);
+        context.currentProperty = _property;
```

Comment:
Rather than setting the currentProperty separately each time it should be added as an argument to the constructor.